templates/gtk/template.json

```diff
@@ -6,13 +6,5 @@
             "default": "/${PROJECT_NAME}/\\w+/\\L\\0\\E/\\W+/-/",
             "pattern": "^[[:word:]-]+$"
         }
-    },
-    "inputs": [
```

Comment:
See other comment. This change needs to be reverted.

Reply:
Fixed